coriolis/conductor/rpc/server.py

```diff
@@ -1137,6 +1137,7 @@ def update_replica(
             # so we must operate on a copy:
             inst_info_copy = copy.deepcopy(replica.info[instance])
             inst_info_copy.update(properties)
+            inst_info_copy['force'] = force
```

Comment:
This has the pretty big unintended side-effect of that 'force' param staying in the Replica's info until further executions.
Indeed it's overridden during Replica updates each time, but maybe some day normal Replica executions will also 'heed' the 'force' param in the task info.
I have no good solution in mind except explicitly setting 'force=False' (at least for the params) before each non-update (which is bad), but for now we should at the very least add a comment saying that force=X will linger in the task_info after the update.